### DISCUSSION TITLE
What are "Custom Arguments"?

### DISCUSSION THREAD

**Discussion by superange128** [micah686/VnManager#27] [Q&A] [answered]
When you import a VN, I see an "Arguments" textbox but I have no idea what this does.

**Comment by micah686** [comment, marked as answer]
That's used if there are custom arguments that are needed to be passed to an exe. So like text hooking with AGTH or something similar might use some arguments. So if the actual exe is named `Test.exe`, the exe + arguments would be like `Text.exe /silent /path="C:\AGTH.exe" /code 15`. So for the custom arguments text box, you would put in: `/silent /path="C:\AGTH.exe" /code 15`, and it would add those arguments when launching the application.

**Reply by micah686** [reply]
Did that answer your question?

**Reply by superange128** [reply]
Yes, thank you.
